### PR TITLE
Enable replay tests for GitLab CI

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

File: src/umpire/util/Macros.hpp [hunk 1]
```diff
@@ -86,7 +86,7 @@
 
 #define UMPIRE_UNUSED_ARG(x)
 
-#define UMPIRE_USE_VAR(x) static_cast<void>(x)
+#define UMPIRE_USE_VAR(x) (void)(x)
```
Comment (mcfadden8) [inline src/umpire/util/Macros.hpp:89]: Is this change still needed?

Comment (davidbeckingsale) [reply src/umpire/util/Macros.hpp:89]: Nope, good point.

**Comment by davidbeckingsale** [comment]
LGTM

File: src/umpire/util/Macros.hpp (hunk 1 above)
Comment (davidbeckingsale) [reply src/umpire/util/Macros.hpp]: Reverted.

**Review by mcfadden8 — APPROVED** [review]

**Review by kab163 — APPROVED** [review]

**Comment by kab163** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
LGTM
